pkg/preparer/orchestrate_test.go

```diff
@@ -565,3 +567,17 @@ func TestPreparerWillAcceptSignatureWhenKeyringIsNil(t *testing.T) {
 		"expected the preparer to verify the signature when no keyring given",
 	)
 }
+
+func TestPrepareHooksFailureRequired(t *testing.T) {
```

Comment:
Is this an unrelated test that is being added for coverage? Was there context for this?

Reply:
@esuen This is for coverage at this point. I found the `if err != nil..` path untested at https://github.com/square/p2/blob/master/pkg/preparer/orchestrate.go#L290. I guess I also just left this in because I initially thought I would need to add the conditional for required hooks in this function. I can remove it if it seems unnecessary now.

Reply:
It's fine, can leave this.